.github/workflows/update-deb-package-snapshots.yml

```diff
@@ -0,0 +1,26 @@
+name: update-snapshots
+on:
+  schedule:
+    - cron:  '0 0 * * *'
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+        
+    - name: Setup Bazel
+      uses: abhinavsingh/setup-bazel@v3
+      with:
+        version: 3.2.0
+
+    - name: Run update script
+      run: ./updateWorkspaceSnapshots.sh
+    
+    - name: Create Pull Request
+      uses: peter-evans/create-pull-request@v2
+      with:
```

Comment:
Do you want to set an author and committer? https://github.com/peter-evans/create-pull-request#committer-and-author

Reply:
Oh, that's interesting. But I think it makes sense to have the author as the bot.